### PR TITLE
Adding iat to the ConsentTokenPayload

### PR DESCRIPTION
**Pull request by banzzai**
## Related Issues

- Solves: T-28748

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

**Review by MadDataScience — CHANGES_REQUESTED** [review]
Don't forget to bump the version here: https://github.com/transcend-io/airgap.js-types/blob/88944ca1d5cdb8632d56e3a0e8c19176e64ebaa7/package.json#L5

**Comment by banzzai** [comment]
> Don't forget to bump the version here:
> 
> https://github.com/transcend-io/airgap.js-types/blob/88944ca1d5cdb8632d56e3a0e8c19176e64ebaa7/package.json#L5

Oh I had that, didn't push :D

**Review by MadDataScience — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/airgap.js-types",
   "description": "TypeScript types for airgap.js interoperability with custom consent UIs",
-  "version": "10.5.1",
+  "version": "10.6.1",
```
Comment (MadDataScience) [inline package.json:5]: nit: either `10.5.2` (since this doesn't really change anything consequential) or `10.6.0`, but not `10.5.1`

**Review by MadDataScience — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (banzzai) [reply package.json:5]: Yeah, not the first time I made that mistake 🙏🏻

**Review by michaelfarrell76 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/airgap.js-types",
   "description": "TypeScript types for airgap.js interoperability with custom consent UIs",
-  "version": "10.6.0",
+  "version": "10.5.2",
```
Comment (MadDataScience) [inline package.json:5]: It looks like you'll need to bump again.

**Review by MadDataScience — CHANGES_REQUESTED** [review]

**Review by MadDataScience — APPROVED** [review]
🚀

**Pull request merged**